src/common/options.rs

```diff
@@ -2286,15 +2332,86 @@ impl AsRef<str> for Language {
     }
 }
 
+impl From<String> for Language {
+    fn from(value: String) -> Language {
+        match &*value {
+            "bg" => Self::bg,
+            "ca" => Self::ca,
+            "cs" => Self::cs,
+            "da" => Self::da,
+            "de" => Self::de,
+            "de-CH" => Self::de_CH,
+            "el" => Self::el,
+            "en" => Self::en,
+            "en-AU" => Self::en_AU,
+            "en-GB" => Self::en_GB,
+            "en-IN" => Self::en_IN,
+            "en-NZ" => Self::en_NZ,
+            "en-US" => Self::en_US,
+            "es" => Self::es,
+            "es-419" => Self::es_419,
+            "es-LATAM" => Self::es_LATAM,
+            "et" => Self::et,
+            "fi" => Self::fi,
+            "fr" => Self::fr,
+            "fr-CA" => Self::fr_CA,
+            "hi" => Self::hi,
+            "hi-Latn" => Self::hi_Latn,
+            "hu" => Self::hu,
+            "id" => Self::id,
+            "it" => Self::it,
+            "ja" => Self::ja,
+            "ko" => Self::ko,
+            "ko-KR" => Self::ko_KR,
+            "lv" => Self::lv,
+            "lt" => Self::lt,
+            "ms" => Self::ms,
+            "nl" => Self::nl,
+            "nl-BE" => Self::nl_BE,
+            "no" => Self::no,
+            "pl" => Self::pl,
+            "pt" => Self::pt,
+            "pt-BR" => Self::pt_BR,
+            "ro" => Self::ro,
+            "ru" => Self::ru,
+            "sk" => Self::sk,
+            "sv" => Self::sv,
+            "sv-SE" => Self::sv_SE,
+            "ta" => Self::ta,
+            "taq" => Self::taq,
+            "th" => Self::th,
+            "th-TH" => Self::th_TH,
+            "tr" => Self::tr,
+            "uk" => Self::uk,
+            "vi" => Self::vi,
+            "zh" => Self::zh,
+            "zh-CN" => Self::zh_CN,
+            "zh-Hans" => Self::zh_Hans,
+            "zh-Hant" => Self::zh_Hant,
+            "zh-TW" => Self::zh_TW,
+            _ => Self::Other(value),
+        }
+    }
+}
+
 impl AsRef<str> for Redact {
     fn as_ref(&self) -> &str {
-        use Redact::*;
-
         match self {
-            Pci => "pci",
-            Numbers => "numbers",
-            Ssn => "ssn",
-            Other(id) => id,
+            Redact::Pci => "pci",
```

Comment:
Redaction supports 50 lower level Entity Types also

https://developers.deepgram.com/docs/supported-entity-types

Reply:
Cool.  Let's create an issue to support those explicitly.  That's out of scope for this PR.

Reply:
(The issue: https://github.com/deepgram-devs/deepgram-rust-sdk/issues/87)